ui/src/components/Claims/util.ts

```diff
@@ -85,24 +51,48 @@ const transformObservation = (obs: Observation<Date, boolean>, linkText: string)
         collapsedText: `${left7.text} / ${right7.text}`,
         children: [left7, right7],
       };
-    case 'DecimalConst':
+    case 'Const':
       return { ...obs, linkText, type: 'Observation', text: obs.value, children: null };
-    case 'DecimalObs':
+    case 'Observe':
       return { ...obs, linkText, type: 'Observation', text: `Price(${obs.value})`, children: null };
-    default:
-      throw new Error('Unknown observation tag: ' + obs.tag);
   }
 };
 
-export const transformClaim = (claim: Claim<Date, Id>, linkText: string): any => {
+export const transformInequality = (
+  inequality: Inequality<Date, string>,
+  linkText: string
+): any => {
+  switch (inequality.tag) {
+    case 'TimeGte':
+      return {
+        ...inequality,
+        linkText,
+        type: 'Observation',
+        children: [], // todo
```

Comment:
Are you planning on implementing this in another PR in the future or as part of this PR ?

Reply:
Thanks, yes I missed this. Having some trouble with this though; I can't get the time parameter to show even if when this is compiling. Dunno if @georg-da D3 expert you can give me a hand with this.